src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -53,13 +53,27 @@
 
 {% block form_row -%}
   {% spaceless %}
+    {% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+    {% set multistoreCheckboxName = 'multistore_' ~ form.vars.name %}
```

Comment:
Maybe add a comment here that this field and `MULTISTORE_FIELD_PREFIX` constant needs to be kept in sync?

Reply:
Also idea of improvement for a later PR, this could be a global variable added in the twig environment config And this config could be linked to the class by using `!const ...` So this way they will necessarily be in sync

Reply:
done !